### PR TITLE
Update FUNDING.yml with Open Collective

### PR DESCRIPTION
**Pull request by lb-**
Will go to https://opencollective.com/wagtail

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 custom: ['https://wagtail.org/sponsor/']
```
Comment (upendrard) [inline .github/FUNDING.yml:1]: wagtail lover

**Comment by lb-** [comment]
Thanks @laymonage

**Comment by laymonage** [comment]
We probably still need to change the repo's settings (I have no access), because I can't see the Sponsor button on the repository.

**Comment by lb-** [comment]
Odd. I thought it was there after the original addition of this file, I'll look into it.

**Comment by lb-** [comment]
Have posted in the core team chat.

**Pull request merged**